src/operator/quantization/dnnl/dnnl_quantize-inl.h

```diff
@@ -67,9 +67,6 @@ static void DNNLQuantizeComputeKer(const std::vector<NDArray>& inputs,
   attr.set_output_scales(mask, scales);
   dnnl::engine cpu_engine = mxnet::CpuEngine::Get()->get_engine();
   NDArray in_buffer       = inputs[0];
-  if (inputs[0].IsView() && inputs[0].IsDNNLData())
```

Comment:
Why have you removed this?

Reply:
Because reorder is handled by the DNNLRun function.